index.html

```diff
@@ -90,6 +92,227 @@ <h2>Let's connect</h2>
+                                </svg>
+
+                            </a>
+                        </li>
+                    </ul>
+
+                    <div class="mt-1">
+                        <a class="btn" href="#">Get my resume</a>
+                    </div>
+
+                </section>
+
+                <ul class="drop-buttons">
+                    <li class="drop-button">
+                        <p class="title">Languages</p>
+                        <ul class="languages extra-menu">
+                            <li>
+                                <img src="../images/javascript.png" alt="javascript icon" />
+                                <p>JavaScript</p>
+                            </li>
+                            <li>
+                                <img src="../images/html.png" alt="javascript icon" />
+                                <p>HTML</p>
+                            </li>
+                            <li>
+                                <img src="../images/css.png" alt="javascript icon" />
+                                <p>CSS</p>
+                            </li>
+                        </ul>
+                    </li>
+
+                    <li class="drop-button">
+                        <p class="title">Frameworks</p>
+                    </li>
+
+                    <li class="drop-button">
+                        <p class="title">Skills</p>
+
+                    </li>
+                </ul>
+            </section>
+
+            <section class="container">
+                <div class="contact-info">
+                    <h1>Contact Me</h1>
```

Comment:
- [ ] Since you have already used `h1` for `Hi, I'm Dagmawi` please consider using other heading tags like `h2`, `h3`, `h4`  .... :pray: